fe/src/contexts/index.ts

```diff
@@ -0,0 +1 @@
+export { HomeContext } from './HomeContext';
```

Comment:
I think you can create this context straight in the `Home` directory. I don't feel like it should be exported as it's not needed by any other components outside of Home

Reply:
Ah good point. 👍